exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -162,13 +164,16 @@ pub fn panic_if_java_options() {
 pub struct JavaServiceFactory;
 
 impl ServiceFactory for JavaServiceFactory {
-    fn command(&mut self, command: &str) -> Option<Box<CommandExtension>> {
-        use exonum::helpers::fabric;
+    fn service_name(&self) -> &str {
+        "JAVA_SERVICE_FACTORY"
```

Comment:
I know that it's definitely not the nicest thing you've ever seen, but I don't think we should try to fix this.